src/sweetalert2.scss

```diff
@@ -758,27 +764,25 @@ body {
   }
 }
 
-.swal2-progresssteps {
-  $lightblue: #add8e6;
-  $blue: #3085d6;
-
+.swal2-progress-steps {
   align-items: center;
   margin: $swal2-progress-steps-margin;
-  padding: 0;
-  font-weight: 600;
+  padding: $swal2-progress-steps-padding;
+  background: $swal2-progress-steps-background;
+  font-weight: $swal2-progress-steps-font-weight;
 
   li {
     display: inline-block;
     position: relative;
   }
 
-  .swal2-progresscircle {
-    width: 2em;
-    height: 2em;
-    border-radius: 2em;
-    background: $blue;
-    color: $swal2-white;
-    line-height: 2em;
+  .swal2-progress-step-circle {
+    width: $swal2-progress-steps-step-width;
```

Comment:
Shouldn't `$swal2-progress-steps-step-width` be named `$swal2-progress-step-circle-width`. That will be consistent with the fact that is the `width` property of the `swal-progress-step-circle` class. 

Similar for the other SCSS variables below this line.

Reply:
Good catch!

Actually, I don't like this legacy assumption about the progress step shape. With `$swal2-progress-steps-step-border-radius: 0;` it's not a circle at all:

![image](https://user-images.githubusercontent.com/6059356/52899764-5d631080-31f6-11e9-93b0-bc3bf14c75bf.png)

https://github.com/sweetalert2/sweetalert2/pull/1411/commits/6c31179d7bc9ab4f276b7934917171842fde1a24